specification/structure.md

```diff
@@ -1,19 +1,19 @@
 ---
 layout: default
-title: Structure of an OpenAPI Document
+title: Structure of an OpenAPI Description
 parent: The OpenAPI Specification Explained
 nav_order: 1
 ---
 
-# Structure of an OpenAPI Document
+# Structure of an OpenAPI Description
 
-An OpenAPI document describes an HTTP-like API in one or more machine-readable files. This page describes the syntax of these files and the minimal structure they must contain.
+An OpenAPI Description (OAD) describes an HTTP-like API in one or more machine-readable documents (files or network resources). This page describes the syntax of these documents and the minimal structure they must contain.
 
-## Document Syntax
+## OpenAPI Description Syntax
 
-An OpenAPI document is a text file, commonly called `openapi.json` or `openapi.yaml`, representing a [JSON](https://en.wikipedia.org/wiki/JSON) object, in either JSON or [YAML](https://en.wikipedia.org/wiki/YAML) format. This file is called the **root document** and it can be split into multiple JSON or YAML files, for clarity.
+OpenAPI Descriptions are written as one or more text documents, commonly called `openapi.json` or `openapi.yaml`, representing a [JSON](https://en.wikipedia.org/wiki/JSON) object, in either JSON or [YAML](https://en.wikipedia.org/wiki/YAML) format. This document is called the **root document**.  An OAD can consist solely of a root document, or it can be split into multiple documents for clarity.
```

Comment:
"one or more text documents" and "commonly called `openapi.json`" is a bit confusing.

 Maybe first introduce the "entry document" and say that "the entry document is commonly called ..."

Reply:
@ralfhandl thanks, will do- this goes better with the "root OpenAPI document" -> "entry OpenAPI document" change in the other PRs, too.